apigw-rest-api-dynamodb-lambda-cdk/README.md

```diff
@@ -0,0 +1,144 @@
+# Amazon API Gateway REST API to Amazon DynamoDB
+
+This pattern creates an Amazon API Gateway REST API that integrates with an Amazon DynamoDB table.
+
+Learn more about this pattern at Serverless Land Patterns: http://serverlessland.com/patterns/apigw-dynamodb-lambda-cdk.
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Architecture
+The following diagram illustrates the solutions architect
+![Architecture Diagram](img/concept.png)
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd apigw-rest-api-dynamodb-lambda-cdk/cdk
+    ```
+
+1. Install dependencies
+    ```
+    npm install
+	cd src
```

Comment:
Should be `cd ..\src`

Reply:
Hey, I am not sure about this. It should be: cd src

Reply:
yes, just cd src . I tested on Mac